.gitignore

```diff
@@ -0,0 +1,11 @@
+# Add any directories, files, or patterns you don't want to be tracked by version control
```

Comment:
.gitignore file must be located in the external folder of the project. In your case, in the same place as README.